app/controllers/curation_concerns/curation_concerns_controller.rb

```diff
@@ -33,6 +33,23 @@ def deny_access(exception)
     end
   end
 
+  def flag
```

Comment:
So technically I can POST a flag even if it's not flaggable. It's not exactly a game-changer, but can we check for it here?